src/python/pants/backend/python/rules/coverage.py

```diff
@@ -295,21 +297,28 @@ async def generate_coverage_reports(
             )
         )
     results = await MultiGet(Get(ProcessResult, PexProcess, process) for process in pex_processes)
+    result_snapshots = await MultiGet(Get(Snapshot, Digest, r.output_digest) for r in results)
+    result_stdouts = tuple(r.stdout for r in results)
```

Comment:
please avoid single letter variables, especially when they conflict with debugger commands.

Reply:
Agreed in general, but these are inside a comprehension. What's the issue when it's in a generator expression like this?